src/cli/serve/__tests__/reload_logging_config.js

```diff
@@ -36,29 +36,48 @@ describe(`Server logging configuration`, function () {
     await primraf(testLogsDirectory);
     await mkdir(testLogsDirectory);
 
-    const conf = await setLoggingDest(`logs/first.log`);
+    let json = Infinity;
+    const conf = setLoggingJson(true);
     const child = spawn(`./bin/kibana`, [`--config`, testConfigFile]);
 
-    // wait for just a bit
-    await wait(5000);
+    child.stdout
+      .pipe(es.split())
+      .pipe(es.mapSync(function (line) {
+        if (!line) {
+          return line; // ignore empty lines
+        }
+        if (json--) {
+          parseJsonLogLine(line);
+        } else {
+          expectPlainTextLogLine(line);
+        }
+      }));
 
```

Comment:
Make sure to handle errors/success properly. This test will always succeed because the test doesn't wait for the client to close or catch assertion errors thrown in `expectPlainTextLogLine()`